test/support/build.js

```diff
@@ -4,6 +4,7 @@
 exports.buildRecord = (record) => {
   return {
     eventSource: 'aws:kinesis',
+    eventSourceARN: 'arn:aws:kinesis:us-east-1:12345678:stream/the-stream-name',
```

Comment:
env var?

Reply:
oh, this is leftover from when i thought the dynamodb would output `postbytes` records onto the same stream it was invoked from.

but now, it invokes on `dovetail-ddb-production`, and outputs on `dovetail-metrics-production`.  So it needs to be an ENV anyways.  I'll clean that up.

Reply:
cool - looks like that is the only change  - after it is committed, I say merge and go